### ISSUE TITLE
Manually start prow jobs

### ISSUE THREAD

**Issue by spiffxp** [kubernetes/test-infra#3336]
I went ahead and created this since it wasn't clear to me if we've met the intent.  Please close with a link to the pertinent issue if this has already been done / is a dupe, or fill out context if there's more to be done.

@spxtr
>This is done for anyone with access to the cluster, which is just google engprod for now.

 @fejta 
>Isn't the goal here to allow someone to click a button on gubernator and/or the prow config?

**Comment by krzyzacy** [comment]
isn't this what mkpj does?

**Comment by spxtr** [comment]
> isn't this what mkpj does?

Yes, but to use it you need the ability to `kubectl create` a resource on the k8s cluster where prow is running.

I'm not particularly interested in implementing some sort of authentication layer so that, for instance, PR authors can rerun their own jobs.

**Comment by spiffxp** [comment]
are we still interested in doing this?

**Comment by BenTheElder** [comment]
`/retest` / `/test foo` should cover most, but not all use cases..

**Comment by stevekuznetsov** [comment]
We also have the [`manual-trigger`](https://github.com/kubernetes/test-infra/tree/master/experiment/manual-trigger) tool which entirely bypasses `prow`.
